ipatests/test_integration/test_automount_locations.py

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+#
+
+"""
+This module provides tests for the automount location feature.
```

Comment:
Style nit pick, the subject belongs on the first line:

```
"""This module provides tests for the automount location feature.
"""
```

Reply:
Fixed, thanks.